packages/extension/src/components/pages/SubmitBulkTransactions/RecapView/RecapView.tsx

```diff
@@ -26,6 +24,9 @@ export const RecapView: React.FC<RecapProps> = ({
   handleReject,
   beginProcess
 }) => {
+  const [isDataExpanded, setIsDataExpanded] = useState(false);
+  const [isFeeExpanded, setIsFeeExpanded] = useState(false);
+
   const getTransactionCountsByType = () => {
```

Comment:
Not related to this MR but it seems that this should be wrapped within a useMemo